lib/src/r_tree/leaf_node.dart

```diff
@@ -22,7 +22,20 @@ class LeafNode<E> extends Node<E> {
   List<RTreeDatum<E>> _items = [];
```

Comment:
How about making this `final`?

Reply:
sure

Reply:
I can make this change, we'll have to change the clearChildren below to use .clear and there's at least one spot where we'll have to duplicate the children so we don't hit concurrent modification errors.